BedrockServer.cpp

```diff
@@ -395,6 +395,14 @@ void BedrockServer::sync()
                 continue;
             }
 
+            if (command->shouldPostProcess()) {
+                BedrockCore::RESULT postProcessResult = BedrockCore::RESULT::INVALID;
+                postProcessResult = core.postProcessCommand(command);
+                if (postProcessResult != BedrockCore::RESULT::COMPLETE) {
```

Comment:
Shouldn't need this block since `postProcessCommand` really doesn't need to return anything.